tests/zkevm/test_worst_compute.py

```diff
@@ -741,6 +741,101 @@ def test_worst_unop(
     )
 
 
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("key_mut", [True, False])
+@pytest.mark.parametrize("val_mut", [True, False])
+def test_worst_tload(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    key_mut: bool,
+    val_mut: bool,
+):
+    """Test running a block with as many TLOAD calls as possible."""
+    env = Environment()
+
+    start_key = 41
+    code_key_mut = Bytecode()
+    code_val_mut = Bytecode()
+    if key_mut and val_mut:
+        code_prefix = Op.PUSH1(start_key) + Op.JUMPDEST
+        code_key_mut = Op.POP + Op.GAS
+        code_val_mut = Op.TSTORE(Op.DUP2, Op.GAS)
```

Comment:
Had to check, but this looks correct: this will store `Op.GAS` at the key which we just mutated at `code_key_mut` (so the key written to is the key we are going to also `TLOAD` in the loop and the value will be a different value than the key)